src/main/resources/com/google/api/codegen/metadatagen/java/grpc/grpc_package.snip

```diff
@@ -0,0 +1,181 @@
+@snippet generate(metadata)
+  buildscript {
+    repositories {
+      mavenCentral()
+    }
+    dependencies { classpath 'com.google.protobuf:protobuf-gradle-plugin:0.7.7' }
+  }
+
+  apply plugin: 'java'
+  apply plugin: 'maven'
+  apply plugin: 'signing'
+  apply plugin: 'com.google.protobuf'
+
+  description = 'GRPC library for {@metadata.identifier}'
+  group = "com.google.api.grpc"
+  version = "{@metadata.packageVersionBound.lower}"
+  sourceCompatibility = 1.6
+  targetCompatibility = 1.6
+
+  repositories {
+    mavenCentral()
+    mavenLocal()
+  }
+
+  dependencies {
+    compile "com.google.protobuf:protobuf-java:{@metadata.protoVersionBound.lower}"
+    compile "com.google.api:api-common:{@metadata.apiCommonVersionBound.lower}"
+  }
+
+  ext {
+    packageName = '{@metadata.identifier}'
+  }
+
+  ext.locateGapicPluginMethod = { ->
+    def outstream = new ByteArrayOutputStream()
+    exec {
+      commandLine 'which', 'gapic_plugin.py'
+      standardOutput = outstream
+    }
+    return outstream.toString().trim()
+  }
+
+  protobuf {
+    protoc {
+      // The version of protoc must match protobuf-java. If you don't depend on
+      // protobuf-java directly, you will be transitively depending on the
+      // protobuf-java version that grpc depends on.
+      artifact = "com.google.protobuf:protoc:{@metadata.protoVersionBound.lower}"
+    }
+    plugins {
+      grpc {
+        artifact = 'io.grpc:protoc-gen-grpc-java:{@metadata.grpcVersionBound.lower}'
+      }
+    }
```

Comment:
You are missing the gapic plugin:

https://github.com/googleapis/packman/blob/master/templates/java/build-alt.gradle.mustache#L58